3p/integration.js

```diff
@@ -337,7 +335,10 @@ function triggerResizeRequest(width, height) {
 }
 
 function triggerRenderStart() {
-  nonSensitiveDataPostMessage('render-start');
+  if (!window.context.isRenderStartSent) {
+    nonSensitiveDataPostMessage('render-start');
+    window.context.isRenderStartSent = true;
```

Comment:
can we add a test case like:

trigger render start via draw3p()
manual call renderStart()
verify message sent only once